android/src/app/organicmaps/widget/placepage/PlacePageLinksFragment.java

```diff
@@ -101,7 +101,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
     mEmail = mFrame.findViewById(R.id.ll__place_email);
     mTvEmail = mFrame.findViewById(R.id.tv__place_email);
-    mEmail.setOnClickListener((v) -> openUrl(Metadata.MetadataType.FMD_EMAIL));
+    mEmail.setOnClickListener((v) -> openMail(Metadata.MetadataType.FMD_EMAIL));
```

Comment:
Should FMD_EMAIL be passed as a parameter? Do we have other emails in the Place Page (PP)?

Reply:
We dont have others e-mail in place page
Do you want i add FMD_EMAIL in openMail without parameter ?

Reply:
```suggestion
    mEmail.setOnClickListener((v) -> Utils.sendTo(requireContext(), getLink(Metadata.MetadataType.FMD_EMAIL)));
```